resolwe/flow/models/fields.py

```diff
@@ -110,8 +110,11 @@ def pre_save(self, instance, add):
                 slug = ''
 
         if slug:
-            # Make sure that potentially added sequence won't excede maximal length.
-            slug = slug[:(self.max_length - MAX_SLUG_SEQUENCE_DIGITS - 1)]
+            # Make sure that auto generated slug with added sequence
+            # won't excede maximal length.
+            # Validation of predefined slugs is handled by Django.
+            if not predefined_slug:
```

Comment:
Can you combine with the if above?

Reply:
No, just trimming has to be skipped for predefined slugs. The rest of the above if is still needed to determine availability of the slug

Reply:
Ah I didn't see there is code below in the same if, nevermind.